src/libs/events.js

```diff
@@ -121,6 +121,13 @@ export const extractCrossWorkspaceDetails = (fromWorkspace, toWorkspace) => {
   }
 }
 
+export const extractBillingDetails = billingProject => {
+  return {
+    billingProjectName: billingProject.projectName,
+    billingProjectCloudPlatform: _.toUpper(billingProject.cloudPlatform) // Should already be uppercase, but enforce for consistency.
```

Comment:
In the workspace case we do need to uppercase (see https://github.com/DataBiosphere/terra-ui/pull/3619).

Reply:
Perhaps rename to `extractBillingProjectMetrics` so it's clear these keys are metric names (unless of course this is used elsewhere)?

Reply:
This is already in `events.js`, which is all in support of MixPanel events, so I think the context is clear. There are also other "extract" methods in this file that don't including metrics in the name (`extractWorkspaceDetails`, for example).